frontend/.env.example

```
@@ -4,4 +4,6 @@ NEXT_PUBLIC_WORKOS_REDIRECT_URI="http://localhost:3000/auth/callback"
 
 # openssl rand -base64 32
 WORKOS_COOKIE_PASSWORD=""
-DATABASE_URL=""
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)